src/server/routes/editor.js

```diff
@@ -112,7 +111,7 @@ router.post('/edit/handler', auth.isAuthenticatedForHandler, (req, res) => {
 		)
 		.then((editor) => {
 			let editorTitleUnlock;
-			if (req.body.title === 'none') {
+			if (req.body.title === 'NULL' || req.body.title === '') {
```

Comment:
I've been thinking for a while, this would be better if we could return null from the form, and then just do:

    if (!req.body.title) { ... }